18_regexp_lab/03.py

```diff
@@ -10,5 +9,12 @@
 
 with open(kuku.path, 'r') as f:
     reader = csv.reader(f)
+    num_cols = len(next(reader))
+    f.seek(0 ,0)
+
     for row in reader:
-        print ('{} {} {}'.format(row[1], row[0], row[2]) )
+        if num_cols > 1:
+            row[0],row[1] = row[1],row[0]
+            print(row)
+        else:#print as is...
+            print(row)

```

Comment:
Seems good, but please don't rewind the file using seek. That should be used only for really low level libraries.